.gitignore

```diff
@@ -1,11 +1,9 @@
 # PhpStorm Settings
-.idea
-# Php Test
-.phpunit.result.cache
-
+/.idea
+# Environment
+/vendor
 # Easy Coding Standard
-.ecs_cache
+/.ecs_cache
 
-# Environment
-composer.lock
```

Comment:
Should always include the `composer.lock` file for easy composer installation and to avoid drastic vendor changes